composer_local_dev/docker_files/entrypoint.sh

```diff
@@ -19,8 +19,9 @@ set -xe
 init_airflow() {
   sudo chown airflow:airflow airflow
 
-  mkdir -p ${AIRFLOW__CORE__DAGS_FOLDER}
-  mkdir -p ${AIRFLOW__CORE__PLUGINS_FOLDER}
+mkdir -p ${AIRFLOW__CORE__DAGS_FOLDER}
+mkdir -p ${AIRFLOW__CORE__PLUGINS_FOLDER}
+mkdir -p ${AIRFLOW__CORE__DATA_FOLDER}
```

Comment:
These lines are now inside the `init_airflow` method, please indent them accordingly.